### PR TITLE
update psh and prsync test cases

### PR DESCRIPTION
**Pull request by caomengmeng**
@tingtli please help to review. 
All cases are passed in redhat test environment.

### CODE REVIEW

File: xCAT-test/autotest/testcase/prsync/cases0 [hunk 1]
```diff
@@ -34,6 +40,8 @@ check:rc==0
 end
 
 start:prsync_file_node
+description: run prsync -o to copy file from MN to CN
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/prsync/cases0]: Where do you use -o in this case?

File: xCAT-test/autotest/testcase/prsync/cases0 [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 start:prsync_v
+description: prsync -v
+os:Linux
 cmd:prsync -v
 check:rc==0
 check:output=~Version
 end
 
 start:prsync_dir_node
+description: run prsync -o to copy files in dir from MN to CN
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/prsync/cases0]: I dont think this case could run successfully. See Line 26.
What does " check:$$CN: done" mean?

File: xCAT-test/autotest/testcase/prsync/cases0 (hunk 1 above)
Comment (tingtli) [inline xCAT-test/autotest/testcase/prsync/cases0]: same problem with case prsync_dir_node. see line check:$$CN: done

File: xCAT-test/autotest/testcase/psh/cases0 [hunk 3]
```diff
@@ -8,25 +10,25 @@ check:output=~Usage
 end
 
 start:psh_v
+description: psh -v
+os:Linux
 cmd:psh -v
-check:rc==0
-end
-
-start:psh_i
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/psh/cases0]: we could not delete this case since -i is an important option for psh

**Comment by tingtli** [comment]
A general comment is we need to add a psh -l case .

File: xCAT-test/autotest/testcase/prsync/cases0 (hunk 2 above)
Comment (caomengmeng) [inline xCAT-test/autotest/testcase/prsync/cases0]: It passed as this command in line 26 is ignored by xcattest

**Comment by caomengmeng** [comment]
@tingtli 
please help to review code
1. add psh_i and psh_l cases
2. update checkout:$$CN to be checkout:output=~$$CN
3. update descriptions

**Comment by tingtli** [comment]
@caomengmeng  I don't have further comments for th test cases. Please update bundle files. Thanks!
@immarvin  Please also take a look. Thanks!

**Comment by immarvin** [comment]
i am ok with the changes

**Comment by tingtli** [comment]
i agree to merge.

**Pull request merged**